simplehub_test.go

```diff
@@ -0,0 +1,210 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package pubsub_test
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/juju/testing"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/pubsub"
+)
+
+type SimpleHubSuite struct {
+	testing.LoggingCleanupSuite
+}
+
+var (
+	_ = gc.Suite(&SimpleHubSuite{})
+
+	veryShortTime = time.Millisecond
```

Comment:
This gets used as a timeout for both cases where the tests expect something to happen and when they don't expect something to happen. This seems like a probably cause for unreliable tests.

How about using 2 values like we do in Juju? One shorter one for when we don't expect something to happen, and a long one for when we do.